packages/client/src/object/AggregateOptsThatErrors.ts

```diff
@@ -15,9 +15,25 @@
  */
 
 import type { ObjectOrInterfaceDefinition } from "@osdk/api";
+import type { SingleKeyObject } from "type-fest";
 import type { AggregateOpts } from "../query/aggregations/AggregateOpts.js";
+import type { UnorderedAggregationClause } from "../query/aggregations/AggregationsClause.js";
 import type { GroupByClause } from "../query/aggregations/GroupByClause.js";
 
+export type AggregateOptsThatErrorsAndDisallowsOrderingWithMultipleGroupBy<
```

Comment:
I tried a few different approaches for enforcing the "can't order on $select if there are multiple $groupBy clauses" restriction in the type system, and this method of adding another wrapped type layer around the `AggregateOptsThatErrors` type just to check this restriction was the easiest way to do it. 

@ericanderson was this an okay approach?